core.js

```diff
@@ -253,6 +259,46 @@ function create_patch_request(ch, pull) {
 }
 
 /*
+Replace individual constants in formulae with actual values
+and supply a result.
+
+If pull argument is missing, test arithmetic expression using a default value
+and try to evaluate expression. Returns null if failed.
+
+IL - Inserted lines
+RL - Removed lines
+FC - Files changed
+R  - Rating
+*/
+exports.eval_formulae = function(formulae, pull) {
```

Comment:
This function evaluates the formulae and computes the score. Naming police doesn't agree with this.